freeCodeCamp/freeCodeCamp
# feat: a/b test lock icon

Checklist:






Closes #XXXXX



This pr adds a security lock icon to the donation description and the donation cta. 

How to test:
A variant: seed the db, login, check the email (foo@bar.com), and pop the donation modal. It should not include lock icons.
B variant: seed the db, login, change the email to (foo@foo.com), and pop the donation modal It should have two lock icons.

B variant screenshot of the donation modal:

<img width="982" alt="Screen Shot 2022-03-22 at 10 40 57 AM" src="https://user-images.githubusercontent.com/4591597/159436260-fbe556ad-0570-4d78-91ba-f46ed56002dd.png">
